hardware/microwave/mw_source_smiq.py

```diff
@@ -53,8 +62,13 @@ def on_activate(self):
         # trying to load the visa connection to the module
         self.rm = visa.ResourceManager()
         try:
-            self._gpib_connection = self.rm.open_resource(self._gpib_address,
-                                                          timeout=self._gpib_timeout)
+            if not self._gpib_baud_rate:
```

Comment:
Please use `is not None` here.

Reply:
No worries @drogenlied, I have just made that change now in commit 0c48c57.